### ISSUE TITLE
[Website - Branding Resources on Swag ] Simplifying/ scaling community self printing MP t-shirt 

### ISSUE THREAD

**Issue by rstjames** [microprofile/microprofile-marketing#792]
I was thinking. How can we make it as easy as possible for MP contributors to purchase on their own MP swag and still find a way to control the branding/ quality?

https://teespring.com/ (there are many more similar sites we can pick that maybe be best for US and EU) 

With this site you can upload designs (create mockups), decide colors, material it can be printed on, and manage how much you would like to charge (can set it to no profit). 

Pretty much people can order what they want, how many they want and they will print it and send it to them. But having some control over the branding. 

How it works: 
https://community.teespring.com/training-center/how-does-teespring-work/

Thoughts?

Ryan

**Comment by rstjames** [comment]
I will look into what other sites that are similar Teespring.com is in the US and I am sure I can find a similar setup depending on location

**Comment by rstjames** [comment]
@aeiras

**Comment by rstjames** [comment]
Maybe best way around all these logo discussions, allow only to print the front of the shirt with the MP logo

**Comment by aeiras** [comment]
@rstjames, 

The hoody version of the 2023 design is missing from the drive. https://drive.google.com/drive/u/0/folders/10HK2v2NVrploWgl66jE1o8B6YnYVAGc2

Can you please create those files and add them to the drive? Thanks!

**Comment by rstjames** [comment]
Hoodie updated on the drive @aeiras

**Comment by aeiras** [comment]
@rstjames it looks beautiful. Thank YOU!

Can you find a way to create a landing button under the Branding page so that MPWG Members **can find** all relevant printing swag data for shirt and hoody folders available via 1 link?  That link will be added under the Onboarding wiki, MPWG page, and the such as well.  Doing so will close this ticket. 

A new ticket on "self-printing" ticket is needed to enable individuals to print directly via the website with a trusted vendor, as discussed years ago.  TT is likely to volunteer to execute such a task in Q4 2023.

**Comment by rstjames** [comment]
@aeiras A swag  section was added to the resource page. This give contributors access to the source data for printing MP swag on their own. This is a step before we set up a simple for people to order swag off a website. 
https://microprofile.io/resources/#swag

As for  "self-printing"  I am going over these companies and seeing which ones are in the US and Europe
https://ecommerce-platforms.com/articles/best-print-on-demand-companies-in-europe

From there I think if we pick one we will need to do some test orders. To make sure we select the correct quality shirt/ color and price range.

**Comment by rstjames** [comment]
Updated both tickets.
Step one making the print data available ,
step two setting things up on a self print website.
Step Three test the website.
Step 4 (optional) integrate the shop onto our website. Making it so the visitor doesnt need to travel to the 3rd party site to buy what they want.

**Comment by aeiras** [comment]
@r

> 

@rstjames, 

I love how you organized the files and created the landing link on the branding page. Very efficient to navigate upon arriving at the branding page, exactly what was needed.   **How about adding the USA vendor folder to that space as well?**  the Link is below. I don't wish to touch the folder, thank you. 

https://drive.google.com/drive/folders/1WVewzWuskwyWD_7ATQeEf_iHjERxAWCD

**Comment by aeiras** [comment]
> Updated both tickets.
> Step one making the print data available ,
> step two setting things up on a self print website.
> Step Three test the website.
> Step 4 (optional) integrate the shop onto our website. Making it so the visitor doesn't need to travel to the 3rd party site to buy what they want.

+ 1 to all steps with 1 completed completed, the 2, 3 & 4 required research.  There is no rush on the completion of the task as we currently must complete the Projects' landing pages first.  ETA for swag improvement could be Q4 based on current priorities. Yet, if stuff can be done in parallel, that is also a positive. 

You rock, Triber!

**Comment by rstjames** [comment]
@aeiras needs to follow up with Apache software infra Swag setup.
